homeassistant/components/firmata/const.py

```diff
@@ -0,0 +1,23 @@
+"""Constants for the Firmata component."""
+import logging
+
+LOGGER = logging.getLogger(".")
```

Comment:
`.` will just use `.` as the name.
```suggestion
LOGGER = logging.getLogger(__package__)
```